resolwe_bio/tests/workflows/test_rna_seq.py

```diff
@@ -107,24 +107,19 @@ def test_custom_cutadapt_star_htseq_workflow(self):
     @tag_process('workflow-custom-cutadapt-star-rsem-single', 'workflow-custom-cutadapt-star-rsem-paired')
     def test_custom_cutadapt_star_rsem_workflow(self):
         with self.preparation_stage():
+            single_reads = self.prepare_reads(['reads rsem.fq.gz'])
+            paired_reads = self.prepare_paired_reads(mate1=['reads rsem.fq.gz'], mate2=['reads rsem2.fq.gz'])
+            annotation = self.prepare_annotation('annotation_rsem.gtf.gz', source='ENSEMBL')
```

Comment:
Is the custom `source` still required?

Reply:
I think so (`def prepare_annotation(self, fn='sp_test.gtf', source='DICTYBASE')`).

Reply:
The test runs correctly with the default `source` as well. Custom source can be removed.